terraform/modules/stack/network.tf

```diff
@@ -1,7 +1,7 @@
-module "interface_endpoints" {
+/*module "interface_endpoints" {
   source = "../interface_endpoints"
 
   vpc_id            = var.vpc_id
   security_group_id = aws_security_group.egress.id
   subnet_ids        = var.private_subnets
-}
+}*/
```

Comment:
Should we just remove this? If we're keeping it, what for?

Reply:
Yeah, it should be removed. Done.